sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_integers.py

```diff
@@ -162,7 +162,10 @@ def __str__(self):
     def to_byte_array(self):
         high_bytes = self.high.value.to_bytes(8, byteorder='little')
         low_bytes = self.low.value.to_bytes(8, byteorder='little')
-        return low_bytes + high_bytes
+        byte_array = bytearray()
```

Comment:
Just out of curiosity - wouldn't it be more efficient to allocate the byte array initially with the right size already like bytearray(high_bytes.length() + low_bytes.length()) - or size - whatever is the best way toget the size of the sprevious lists/arrays?